src/Uno.UWP/Devices/Sensors/Extensions/LongExtensions.Android.cs

```diff
@@ -0,0 +1,20 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.OS;
+using Java.Lang;
+
+namespace Windows.Devices.Sensors.Extensions
+{
+	public static class LongExtensions
+	{
+		public static DateTimeOffset SensorTimestampToDateTimeOffset(this long timestamp)
```

Comment:
Extensions on primitive types should be internal, not public.

Should be a helper, not an extension.

Reply:
True, will fix